examples/dlq/main.tf

```diff
@@ -0,0 +1,18 @@
+data "aws_kms_key" "sns" {
+  key_id = "alias/ops/sns"
+}
+
+module "sqs_queue" {
+  source    = "../../"
+  name      = "test-queue-naveen"
+  topic_arn = "arn:aws:sns:us-east-1:account_num:dpl-test-topic-naveen"
+  dlq_options = {
```

Comment:
Can you give examples of all `dlq_optioins`

Reply:
```redrive_policy: {} <-- should be json encoded in the module
      name: string, optional and defaults to ${QUEUE_NAME}_dlq
      reference: string
      max_recieve_count: Integer, not optional, must be greater than 1
      visibility_timeout: Integer, optional, default 30s
      message_retention_seconds: Integer, optional, defaults to whatever 14 days in seconds is```

Reply:
added